libs/build_dod_dataset.py

```diff
@@ -135,6 +256,9 @@ def get_usa_by_county_df():
     # handle serializing FIPS without trailing 0
     final_df['State/County FIPS Code'] = final_df['State/County FIPS Code'].astype(str).str.replace('\.0','')
 
+    # There should be a leading zero (FIPS codes are of length 5). Ideally this would be a char
+    final_df['State/County FIPS Code'] = final_df['State/County FIPS Code'].astype(str).str.rjust(5, '0')
```

Comment:
if you want this behavior I think you should just delete the line above, that way we dont have to do any padding

Reply:
I guess this isn't in this change, but if you do:
```
data = pd.read_csv(file, dtype={"FIPS": str})
data = data['State/County FIPS Code'].str.zfill(5)
```
I think that would work!